prisma/migrations/20220920112540_add_like_model/migration.sql

```diff
@@ -0,0 +1,28 @@
+/*
+  Warnings:
+
+  - You are about to alter the column `password` on the `User` table. The data in that column could be lost. The data in that column will be cast from `Text` to `VarChar(250)`.
+
+*/
+-- AlterTable
+ALTER TABLE "Profile" ALTER COLUMN "profileImageUrl" SET DEFAULT E'https://www.pngfind.com/pngs/m/676-6764065_default-profile-picture-transparent-hd-png-download.png';
+
+-- AlterTable
+ALTER TABLE "User" ALTER COLUMN "password" SET DATA TYPE VARCHAR(250);
+
+-- CreateTable
+CREATE TABLE "Like" (
+    "likedById" INTEGER NOT NULL,
```

Comment:
This migration file is a little odd to have now as we're adding columns, only for the next migration to remove them. Delete these two migration files and run the `npx prisma migrate dev` command to combine them

Reply:
Fixed!